azext_iot/central/models/device.py

```diff
@@ -31,3 +31,14 @@ def _parse_device_status(self) -> DeviceStatus:
             return DeviceStatus.registered
 
         return DeviceStatus.provisioned
+
+    def get_device_registration_info(self):
+        registration_info = {
```

Comment:
I'd also suggest renaming `device_status` to `status` but thats very very nit, so feel free to #wontfix

Reply:
wont fix